packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -157,6 +157,7 @@ export const makeVaultManager = (
         liquidationInProgress = false;
         // XXX should notify interested parties
         console.error('liquidateAndRemove failed with', e);
+        throw e;
```

Comment:
was this necessary or hygiene?

Reply:
I think this was to stop the infinite attempt to re-liquidate something already in progress of liquidation. Since it's state has changed, it will always fail.